core/shared/src/main/scala/org/http4s/Uri.scala

```diff
@@ -428,7 +428,14 @@ object Uri extends UriPlatform {
     def toRelative: Path = copy(absolute = false)
   }
 
-  object Path {
+  private[Uri] trait LowPriorityImplicitsForPath {
+    implicit val pathHash: Hash[Path] = new Hash[Path] {
+      override def hash(x: Path): Int = x.##
+      override def eqv(x: Path, y: Path): Boolean = x == y
+    }
+  }
```

Comment:
Nice! Sorry, on second thought I think we can just mixin `with Hash` to the existing instances.
https://github.com/http4s/http4s/blob/8e31e5f69b1e7b92d7bb3176a8c0719d5b4b7cc2/core/shared/src/main/scala/org/http4s/Uri.scala#L563-L564

Reply:
Right!